### PR TITLE
Sandraschumann

### PR DESCRIPTION
**Pull request by sandra-schumann**
A quick and dirty quick start guide for the README file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,16 @@
 # robotont
 The metapackage for robotont
 
+## Quick Start Guide
```
Comment (RValner) [inline README.md:4]: Maybe rename it to "Setting up the real robot" or something

File: README.md [hunk 2]
```diff
@@ -1,5 +1,16 @@
...
+- Physically build your robot (if necessary), connect a power supply (battery or power adapter) and turn on the robot. You may also need to turn on the onboard computer separately.
+- By default, the robot creates a wireless hotspot with SSID `clearbot-#`, where # is the robot's number, and password `t0ndik00bas`. Connect to it from a device with SSH capability (e.g. a laptop).
+    - Robot's number can be found on a sticker on the front side of the onboard computer.
+- SSH into the robot at `kasutaja@clearbot-#` where # is the robot's number (same as the wireless SSID connected to) and default password `t0ndik00bas`.
+    - If your device is unable to connect to the robot based on hostname, then try connecting based on IP-address: `kasutaja@192.168.200.#` where # is again your robot's number.
+- Launch a demo program using the command `roslaunch robotont_teleop teleop_keyboard.launch` and control the robot using your keyboard.
```
Comment (RValner) [inline README.md:10]: robotont_teleop is now robotont_tutorials

File: README.md [hunk 3]
```diff
@@ -1,5 +1,16 @@
...
+    - For controls, see the terminal output when launching the command.
+
+For a more thorough tutorial for beginners as well as tutorials on other topics see the [Robotont homepage](http://robotont.ut.ee/).
+
 [![Build Status](https://travis-ci.org/robotont/robotont.svg?branch=master)](https://travis-ci.org/robotont/robotont)
```
Comment (RValner) [inline README.md:15]: Keep the badges in front of the readme ;)

**Review by RValner — CHANGES_REQUESTED** [review]
Minor fixes needed, refer to the comments.

**Comment by kruusamae** [comment]
migrated to github.io

**Pull request closed without merging**